### PR TITLE
Refactor test names

### PR DESCRIPTION
**Pull request by justineyster**
Just a small CI refactor

### CODE REVIEW

File: test_config.yaml [hunk 1]
```diff
@@ -14,16 +14,16 @@ fileExistenceTests:
   shouldExist: true
   uid: 1002
   gid: 0
-- name: hive-site.xml
+- name: metastore-site.xml
```
Comment (apreethi13) [inline test_config.yaml:17]: confused why we want to rename `hive-site.xml` to `metastore-site.xml`

Comment (apreethi13) [reply test_config.yaml:17]: can you add more context

Comment (justineyster) [reply test_config.yaml:17]: I'm just renaming the test.

Comment (justineyster) [reply test_config.yaml:17]: We changed the name of the file in #37.

Comment (apreethi13) [reply test_config.yaml:17]: So is `hive-site.xml` renamed to `metastore-site.xml`? Would this change need to  be reflected in our environment too?

**Review by apreethi13 — APPROVED** [review]
👍

**Pull request merged**